code/modules/mob/living/living.dm

```diff
@@ -461,7 +461,7 @@
 		s_active.close(src)
 
 /mob/living/movement_delay(ignorewalk = 0)
-	. = ..()
+	. = 0
```

Comment:
This just stops an unnecessary calculation I think. 👌

Reply:
Yep, called every time a `mob/living` moves I think.